.github/actions/latest-wrangler/main.py

```diff
@@ -30,8 +30,8 @@
             sys.exit(1)
         else:
             # everything is the latest if the package doesn't exist
-            print(f"::set-output name=latest::{True}")
-            print(f"::set-output name=minor_latest::{True}")
+            print(f"latest={True} >> $GITHUB_OUTPUT")
+            print(f"minor_latest={True} >> $GITHUB_OUTPUT")
```

Comment:
I'm not sure this is going to work anymore.  Let's remove your changes to this file for this PR and we will handle it internally.

Reply:
I agree :) I removed the changes.